go-controller/pkg/ovn/ovn.go

```diff
@@ -438,9 +440,12 @@ func (oc *Controller) iterateRetryPods(updateAll bool) {
 	defer oc.retryPodsLock.Unlock()
 	now := time.Now()
 	for uid, podEntry := range oc.retryPods {
+		if podEntry.ignore {
```

Comment:
who sets it back to to ignore = false?

Reply:
oops, i missed doing that in update, thanks